core/templates/services/question-validation.service.ts

```diff
@@ -12,26 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { Injectable } from '@angular/core';
```

Comment:
All the imports can be shifted after the fileoverview.

Reply:
Done.